api/src/clojure/org/akvo/flow_api/unilog/unilog.clj

```diff
@@ -39,21 +39,36 @@
 
 (def parse-json #(json/parse-string % true))
 
-(defn form-data [{:keys [formId id surveyId]}]
-  {:form-id formId
+(defn form-data [{:keys [formId id]}]
+  {:owner-id formId
```

Comment:
If we talk in the collapse logic about _parents_, what do you think on using `:parent-id` instead of `:owner-id`?

Reply:
To be honest, I did not like either of the names.